src/script/builder/tir/prim_func_frame.cc

```diff
@@ -32,12 +33,8 @@ namespace tir {
 
 PrimFuncFrame FindPrimFuncFrame(const String& method) {
   Builder builder = Builder::Current();
-  if (Optional<PrimFuncFrame> prim_func_frame = builder->FindFrame<PrimFuncFrame>()) {
-    if (Optional<BlockFrame> block_frame = builder->GetLastFrame<BlockFrame>()) {
-      if (prim_func_frame.value()->root_block_frame.get() == block_frame.get()) {
-        return prim_func_frame.value();
-      }
-    }
+  if (Optional<PrimFuncFrame> prim_func_frame = builder->GetLastFrame<PrimFuncFrame>()) {
+    return prim_func_frame.value();
```

Comment:
code duplication between the else branch and the lines following the if conditional